docs/reference/index.md

```diff
@@ -66,6 +66,7 @@ mdwrap is a command line tool to format our Markdown documentation made with
 | [summarization](./services/summarization.md)                           | <https://github.com/swiss-ai-center/summarization-service>              | -                                                                        | -                                                             |
 | [text-recognition-ocr](./services/text-recognition-ocr.md)             | <https://github.com/swiss-ai-center/text-recognition-ocr-service>            | <https://text-recognition-ocr-swiss-ai-center.kube.isc.heia-fr.ch>            | <https://text-recognition-ocr-service.swiss-ai-center.ch>     |
 | [text-summarizer](./services/text-summarizer.md)                       | <https://github.com/swiss-ai-center/text-summarizer-service>            | <https://text-summarizer-swiss-ai-center.kube.isc.heia-fr.ch>            | <https://text-summarizer-service.swiss-ai-center.ch>          |
+| [urban-sound-classifier](./services/urban-sound-classifier.md)         | <https://github.com/swiss-ai-center/urban-sound-classifier>             | <https://urban-sound-classifier-swiss-ai-center.kube.isc.heia-fr.ch>      | <https://urban-sound-classifier.swiss-ai-center.ch>           |
```

Comment:
For the production URL, it's https://urban-sound-classifier-service.swiss-ai-center.ch and for the github https://github.com/swiss-ai-center/urban-sound-classifier-service